content/vault/v1.21.x (rc)/content/docs/license/utilization/auto-reporting.mdx

```diff
@@ -13,13 +13,17 @@ description: >-
 Automated license utilization reporting sends license utilization data to
 HashiCorp without requiring you to manually collect and report them.
 
-Automated reporting shares the minimum data required to validate license
-utilization as defined in our contracts. They consist of mostly computed metrics
-and will never contain Personal Identifiable Information (PII) or other
-sensitive information. Automated reporting shares the data with HashiCorp using
-a secure, unidirectional HTTPS API and makes an auditable record in the product
-logs each time it submits a report. The reporting process submits
-reports roughly once every 24 hours.
+Automated reporting shares the minimum data required to validate license utilization
+as defined in our contracts. They consist of mostly computed metrics and will never
+contain Personal Identifiable Information (PII) or other sensitive information.
+As of Vault 1.21+, the metrics include counters related to the number of:
+
+- [Vault clients](../../concepts/client-count/counting)
+- Certificates issued by Vault's built-in PKI secrets engine
+
+Automated reporting shares the data with HashiCorp using a secure, unidirectional
+HTTPS API and makes an auditable record in the product logs each time it submits
+a report. The reporting process submits reports roughly once every 24 hours.
```

Comment:
```suggestion
Automated reporting writes to your Vault logs every time it submits a
report and shares the report data with HashiCorp using a secure,
unidirectional HTTPS API call. The reporting process submits reports
roughly once every 24 hours.
```
Reordered the sentence to bring the pronoun and proper noun closer together in the sentence